### DISCUSSION TITLE
help me to check the DNS records manually

### DISCUSSION THREAD

**Discussion by dev4ma** [postalserver/postal#1967] [Help with using Postal]
Hello i have problem after install on ubuntu 20.04 TLS

Adding the domain is not accepted at all,

![164539956-a16b8e6f-0612-4479-b39b-3e43a738cc86](https://user-images.githubusercontent.com/56938522/164678406-88139ea9-aba7-4a3b-9f23-31d538ee5a48.png)

### I need to clarify how to check the DNS records on the server with the 

> dig 

or 

> nslookup 

commands to verify how your server sees the records.

**Comment by willpower232** [comment]
If you ssh onto your server and do `dig txt over59.online`, what do you see?

**Reply by dev4ma** [reply]
**The problem was resolved by removing the server and setting it up again**

**Thank you**
